src/riot-rs-threads/src/lib.rs

```diff
@@ -153,6 +164,11 @@ impl Threads {
         old_state
     }
 
+    /// Returns the state of a thread.
+    ///
+    /// # Panics
+    ///
+    /// Panics if `thread_id` is >= [`THREADS_NUMOF`].
```

Comment:
See above. Because we check here `is_valid_pid`, I don't think a panic can happen.